tests/integration/components/polaris-action-list-test.js

```diff
@@ -3,7 +3,7 @@ import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { findAll, click, render } from '@ember/test-helpers';
 import buildNestedSelector from '../../helpers/build-nested-selector';
-import MockSvgJarComponent from '../../mocks/components/svg-jar';
```

Comment:
Can we remove the mock SVG jar component now?

Reply:
ayyy!